extractor/src/main/java/org/schabi/newpipe/extractor/MetaInfo.java

```diff
@@ -0,0 +1,74 @@
+package org.schabi.newpipe.extractor;
+
+import java.io.Serializable;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.Nonnull;
+
+public class MetaInfo implements Serializable {
+
+    private String title = "";
+    private String text = "";
```

Comment:
Maybe the format of the text should be pointed out (like with video description)